File: coinflip/coinFlip.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>

int main() {
    srand(time(0)); // Seed for randomness
    int result = rand() % 2; // Random 0 or 1
    if (result == 0)
        std::cout << "Heads" << std::endl;
    else
        std::cout << "Tails" << std::endl;
    return 0;
}

File: coinFlip.go
package main

import (
	"fmt"
	"math/rand"
	"time"
)

func main() {
	rand.Seed(time.Now().UnixNano())
	if rand.Intn(2) == 0 {
		fmt.Println("Heads")
	} else {
		fmt.Println("Tails")
	}
}
